src/olympia/zadmin/tests/test_tasks.py

```diff
@@ -122,13 +122,18 @@ def test_fetch_new_langpack(self, mock_sign_file):
         assert addon._current_version
         assert addon.current_version.version == amo.FIREFOX.latest_version
 
+        file_ = addon.current_version.files.get()
+
         # has_complete_metadata checks license and categories were set.
         assert addon.has_complete_metadata(), addon.get_required_metadata()
-        assert addon.current_version.files.all()[0].status == amo.STATUS_PUBLIC
+        assert file_.status == amo.STATUS_PUBLIC
         assert addon.status == amo.STATUS_PUBLIC
 
-        mock_sign_file.assert_called_once_with(
-            addon.current_version.files.get(), settings.SIGNING_SERVER)
+        # Make sure it has strict compatibility enabled (should be done
+        # automatically for legacy extensions, that includes langpacks)
+        assert file_.strict_compatibility is True
```

Comment:
Modified those tests because `fetch_langpacks` does some fancy stuff with the add-ons it fetches, so I thought it would be safer to ensure that it didn't mess with strict compatibility as set by the `RDFExtractor` code.